var/spack/repos/builtin/packages/unifyfs/package.py

```diff
@@ -16,55 +16,48 @@ class Unifyfs(AutotoolsPackage):
 
     homepage = "https://github.com/LLNL/UnifyFS"
     git      = "https://github.com/LLNL/UnifyFS.git"
-    url      = "https://github.com/LLNL/UnifyFS/releases/download/v0.9.0/unifyfs-0.9.0.tar.gz"
+    url      = "https://github.com/LLNL/UnifyFS/releases/download/v0.9.1/unifyfs-0.9.1.tar.gz"
     maintainers = ['CamStan']
 
     version('develop', branch='dev')
     version('0.9.1', sha256='2498a859cfa4961356fdf5c4c17e3afc3de7e034ad013b8c7145a622ef6199a0')
-    version('0.9.0', sha256='e6c73e22ef1c23f3141646aa17058b69c1c4e526886771f8fe982da924265b0f')
 
     variant('auto-mount', default='True', description='Enable automatic mount/unmount in MPI_Init/Finalize')
     variant('hdf5', default='False', description='Build with parallel HDF5 (install with `^hdf5~mpi` for serial)')
     variant('fortran', default='False', description='Build with gfortran support')
-    variant('mdhim', default='False', description='Enable MDHIM build options')
     variant('pmi', default='False', description='Enable PMI2 build options')
     variant('pmix', default='False', description='Enable PMIx build options')
     variant('spath', default='True', description='Use spath library to normalize relative paths')
 
-    depends_on('autoconf',  type='build')
-    depends_on('automake',  type='build')
-    depends_on('libtool',   type='build')
-    depends_on('m4',        type='build')
-    depends_on('pkgconfig', type='build')
+    depends_on('autoconf',        type='build')
+    depends_on('automake',        type='build')
+    depends_on('automake@1.15:',  type='build', when='@0.9.2:')
```

Comment:
Did you intend to add a `version` entry for `0.9.2`?

Reply:
We technically don't have a v0.9.2 yet, but will soon(ish). I did this in preparation for that, but also for things that apply to installing `unifyfs@develop`.

Reply:
Okay but the `when` syntax here is for a numeric version range so won't pick up the dependency for `develop`.  What you want is:

```suggestion
    depends_on('automake@1.15:',  type='build', when='@0.9.2:,develop')
```

Reply:
That would be here and the other places you're specifying '@0.9.2:'. :wink:

Reply:
Oh, that's good to know. I can add it. 

Is `develop` not considered newer than the versions anymore (i.e., included in `@0.9.2:` but excluded from `@:0.9.2`)?
https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison

Reply:
It's my understanding is `develop` is required but I'll check.

Reply:
@CamStan Turns out you are correct!

Reply:
Thank you for looking into that @tldahlgren. 😄 I was ready to push your change when that section in the docs made me want to at least check first.